examples/meta/src/binary/svmlight_batch_linadd.sg

```diff
@@ -0,0 +1,35 @@
+File f_feats_train = csv_file("../../data/fm_train_dna.dat")
+File f_feats_test = csv_file("../../data/fm_test_dna.dat")
+File f_labels_train = csv_file("../../data/label_train_dna.dat")
+
+#![create_features]
+Features feats_train = string_features(f_feats_train, enum EAlphabet.DNA)
+Features feats_test = string_features(f_feats_test, enum EAlphabet.DNA)
+Labels labels_train = labels(f_labels_train)
```

Comment:
For all examples that use code that is guarded (gpl, svm light) you need to add them to ignore lists in FindMetaExamples.cmake

Reply:
done